js/app.js

```diff
@@ -1,62 +1,77 @@
 'use strict';
 
-console. log('hello world');
+console.log('hello world');
 
-alert ('Welcome to my About Me page!');
+alert('Welcome to my About Me page!');
 
 let usrnm = prompt('What do you like to be called?');
-console. log(usrnm);
-alert('Cheers ' + usrnm+ '!');
+console.log(usrnm);
+alert('Cheers ' + usrnm + '!');
 
 alert('Guesing game: Please answer the following, yes or no, questions.');
 
-let pets = prompt ('Do you think I have any pets?').toLowerCase();
+function havePets() {
+  let pets = prompt('Do you think I have any pets?').toLowerCase();
 
-if (pets === 'yes' || pets === 'y') {
+  if (pets === 'yes' || pets === 'y') {
   /*console. log*/alert('Not at the moment ' + usrnm + 'but quite a few over my lifetime.');
-} else if (pets === 'no' || pets === 'n') {
+  } else if (pets === 'no' || pets === 'n') {
   /*console. log*/alert('Wow ' + usrnm + '. Spot on guess!');
-} else {
+  } else {
   /*console. log*/alert('Please answer with yes or no only. Thanks ' + usrnm + '!');
+  }
 }
+havePets();
```

Comment:
It's a standard practice to put all your function calls at the bottom of a document. That way you can scroll to the bottom and easily see what functions are being called in what order. It's very helpful for bug fixing!